ledger_device_sdk/src/ui/gadgets.rs

```diff
@@ -719,6 +724,98 @@ pub struct Field<'a> {
     pub name: &'a str,
     pub value: &'a str,
 }
+
+impl<'a> Field<'a> {
+    pub fn event_loop(&self, incoming_direction: ButtonEvent) -> ButtonEvent {
+        let mut buttons = ButtonsState::new();
+        let chunk_max_lines = layout::MAX_LINES - 1;
+        let page_count = 1 + self.value.len() / (chunk_max_lines * MAX_CHAR_PER_LINE);
+
+        let mut cur_page = match incoming_direction {
+            ButtonEvent::LeftButtonRelease => page_count - 1,
+            ButtonEvent::RightButtonRelease => 0,
+            _ => 0,
+        };
+
+        // A closure to draw common elements of the screen
+        // cur_page passed as parameter to prevent borrowing
+        let draw = |page: usize| {
+            clear_screen();
+            let mut chunks = [Label::default(); layout::MAX_LINES];
+            for (i, chunk) in self
+                .value
+                .as_bytes()
+                .chunks(MAX_CHAR_PER_LINE)
+                .skip(page * chunk_max_lines)
+                .take(chunk_max_lines)
+                .enumerate()
+            {
+                chunks[1 + i] = Label::from(core::str::from_utf8(chunk).unwrap());
```

Comment:
this will work for ascii strings, but may panic for arbitrary utf8

Reply:
current `"1.4.2"` `ledger_device_sdk` crashes on displaying a string in chinese? with `MultiFieldReview`:
```rust
"《東明聞見錄》.8.9.10"
```
and a Georgian one: 
```rust
გთხოვთ ახლავე გაიაროთ
```
,
but works with a sample string in russian and a German? one:

```rust
"„Anführungszeichen“ .8.9"  
```
displaying blanks instead of stranger characters

Reply:
The SDK only supports ASCII characters anyway and would simply display blanks (or maybe crash) on anything else.
I'm not a fan of unwrapping, but rather than using `unsafe { from_utf8_unchecked() }` and justifying with a comment, I'd rather keep the code panicking on an incorrect string

Reply:
But since it would not panic on unknown alphabets I replaced some of them with `unwrap_or("")`